src/main/java/com/example/expense/tracker/controller/HelloWorldController.java

```diff
@@ -0,0 +1,15 @@
+package com.example.expense.tracker.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+public class HelloWorldController {
```

Comment:
why do you need this controller? Create a new RestController for health check.

Reply:
controller updated as RestController

Reply:
chnaged as RestController